src/time.rs

```diff
@@ -0,0 +1,7 @@
+use core::convert::TryInto;
```

Comment:
Ditto. Let's be careful when extending the API surface with all sorts of utilities, until we have good use cases for those. And ideally - traits in `embedded-svc` or `embedded-hal`.